app/config/AppConfig.scala

```diff
@@ -84,39 +83,31 @@ class AppConfig @Inject()(val config: Configuration, servicesConfig: ServicesCon
   lazy val SDESNotificationInfoType: String = config.get[String]("SDESNotification.informationType")
   lazy val SDESNotificationFileRecipient: String = config.get[String]("SDESNotification.file.recipient")
 
-  def getPenaltyDetailsUrl: String = {
-    if (!isEnabled(CallAPI1812ETMP)) stubBase + "/penalties-stub/penalty/details/VATC/VRN/"
-    else etmpBase + "/penalty/details/VATC/VRN/"
-  }
+  // Non-agnostic 1812
+  def getPenaltyDetailsUrl: String =
+    if (isEnabled(CallAPI1812ETMP)) etmpBase + "/penalty/details/VATC/VRN/"
+    else stubBase + "/penalties-stub/penalty/details/VATC/VRN/"
 
-  def getPenaltyDetailsUrlCorrect: String = {
-    if (!isEnabled(CallAPI1812ETMP)) stubBase + "/penalties-stub/penalty/details/"
-    else etmpBase + "/penalty/details/"
-  }
-  def getFinancialDetailsUrl(vrn: String): String = {
-    if (!isEnabled(CallAPI1811ETMP)) stubBase + s"/penalties-stub/penalty/financial-data/VRN/$vrn/VATC"
+  // Non-agnostic 1811
+  def getFinancialDetailsUrl(vrn: String): String =
+    if (isEnabled(CallAPI1811Stub)) stubBase + s"/penalties-stub/penalty/financial-data/VRN/$vrn/VATC"
     else etmpBase + s"/penalty/financial-data/VRN/$vrn/VATC"
-  }
 
-  def getFinancialDetailsItsaUrl(nino: String): String = {
-    if (!isEnabled(CallAPI1811ETMP)) stubBase + s"/penalties-stub/penalty/financial-data/NINO/$nino/ITSA"
-    else etmpBase + s"/penalty/financial-data/NINO/$nino/ITSA"
-  }
+  // Non-agnostic 1808
+  def getAppealSubmissionURL(penaltyNumber: String): String =
+    if (isEnabled(CallPEGA)) pegaBase + s"/penalty/first-stage-appeal/$penaltyNumber"
+    else stubBase + s"/penalties-stub/penalty/first-stage-appeal/$penaltyNumber"
 
-  def getAppealSubmissionURL(penaltyNumber: String): String = {
-    if (!isEnabled(CallPEGA)) stubBase + s"/penalties-stub/penalty/first-stage-appeal/$penaltyNumber"
-    else pegaBase + s"/penalty/first-stage-appeal/$penaltyNumber"
-  }
-
-  def getRegimeAgnosticAppealSubmissionUrl(penaltyNumber: String): String = {
-    if (!isEnabled(CallPEGA)) stubBase + s"/penalties-stub/penalty/first-stage-appeal/$penaltyNumber"
-    else pegaBase + s"/penalty/first-stage-appeal/$penaltyNumber"
-  }
+  // Agnostic 1808
+  def getRegimeAgnosticAppealSubmissionUrl(penaltyNumber: String): String =
```

Comment:
I realise agnostic but non-HIP urls are pointless since ITSA will only ever use the HIP urls, but this is repercussion to how the agnostic changes were made by duplicating the connectors etc. Would be simpler if we'd had time to do agnostic clean up before HIP changes